bin/node/runtime/src/impls.rs

```diff
@@ -339,13 +370,18 @@ mod tests {
 			100 * kb,
 			mb,
 			10 * mb,
-			Weight::max_value() / 2,
-			Weight::max_value(),
+			2147483647,
+			4294967295,
+			MaximumBlockWeight::get() / 2,
+			MaximumBlockWeight::get(),
+			// TODO these are not safe
+			// Weight::max_value() / 2,
+			// Weight::max_value(),
```

Comment:
This was okay before because `u32::max_value` was close to `1_000_000_000`. But `u64::max_value` is WAY bigger than `2_000_000_000_000`, and stuff breaks down here. See my comment above.

Reply:
I will check this at the very end.